docs/installation.md

```diff
@@ -60,14 +70,17 @@ Now that the database file can be stored somewhere, we can launch the container.
                 - '5656:5656'
             image: 'mrcas/kapowarr:latest'
     ```
+    A few important notes about the command/file before launching the container:
+    
+    1. If you're using a folder on the host machine instead of a docker volume to store the database file, replace `kapowarr-db` with the path to the host folder.  
+    E.g. `/opt/Kapowarr/db:/app/db`.
+    2. Replace `/path/to/download_folder` with the path to your desired download folder on the host machine. Everything is downloaded to this folder and when completed, moved out of to their final destination. It's smart to set this on a disk that can sustain more writes than normal. Ideally something like a _non_-network mounted ssd, but pretty much everything will suffice.
+    3. Replace `/path/to/root_folder` with the path to your desired root folder on the host machine. Then, this folder will get mapped to `/comics-1` inside the docker container. When adding a root folder in Kapowarr, you'll then set it's location as `/comics-1`, mapping it this way to where ever `/path/to/root_folder` may point.
+    4. You can map multiple root folders by repeating `- /path/to/root_folder:/comics-1` , but then supplying different values for `/path/to/root_folder` and `/comics-1`.  
+    E.g. `- /media/Comics-2:/comics-2` for the Docker Compose file, then add `/comics-2` as your second root folder in Kapowarr.
+    5. Information on how to change the port can be found on the [Setup After Installation page](../setup_after_installation/#port).
+    6. Using a named volume in docker requires you to create the volume before you can use it (refer to [Named Volumes](#named-volumes))
 
-A few important notes about the command/file before launching the container:
```

Comment:
Why did you turns this change back? The text is the same for both the docker CLI and docker compose. So why put the text in the box instead of just leaving it below like I had it?

Reply:
While it looks the same at a glance, they do have some _very_ minor differences - namely, where instructing to repeat or duplicate bits for extra folders. The one under compose is formatted as the line to duplicate, and the one under CLI is formatted as the additional `-v` command.